### PR TITLE
EPMFARMATS-12600 ver. 2

### PR DESCRIPTION
**Pull request by overdd**

### CODE REVIEW

File: EPMFARMATS-12600.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+console.log(now); 
+
+//Get the day, month and year of the current date and separately output to the console
+const now = new Date();
+console.log(now.getDate());
+console.log(now.getMonth());
+console.log(now.getFullYear());
+
+//https://learn.javascript.ru/datetime
+//������� ���� � ������� ��.��.��
```
Comment (SiarheiRylach) [inline EPMFARMATS-12600.js:12]: smth is wrong here

File: EPMFARMATS-12600.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+//Get the current date, print to the console
+const now = new Date();
+console.log(now); 
+
+//Get the day, month and year of the current date and separately output to the console
+const now = new Date();
+console.log(now.getDate());
+console.log(now.getMonth());
```
Comment (SiarheiRylach) [inline EPMFARMATS-12600.js:8]: +1, because months starts from 0

File: EPMFARMATS-12600.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+//�������� ������� formatDate(date), ������� ������� ���� date � ������� ��.��.��:
+
+function formatDate(date) {
+    day = date.getDate();
+    month = date.getMonth() + 1;
+    year = date.getFullYear() % 100;
+    if (day < 10) day = "0" + day;
+    if (month < 10) month = "0" + month;
+    if (year < 10) year = "0" + year;
+    alert(day + "." + month + "." + year)
```
Comment (SiarheiRylach) [inline EPMFARMATS-12600.js:22]: Nice. I think you are ready to use template string. 
`${day}.${month}.${year}`.

**Comment by overdd** [comment]
Encoding and conflict are fixed

**Pull request merged**